### DISCUSSION TITLE
Feedback on PropertyLayer and _PropertyGrid

### DISCUSSION THREAD

**Discussion by EwoutH** [mesa/mesa#1932] [General]
We would love to hear what you think about the new PropertyLayer and _PropertyGrid classes!

**Resources**
- https://github.com/projectmesa/mesa/issues/1889
- https://github.com/projectmesa/mesa/pull/1898

A lot of examples are available in the PR (#1898).

**Comment by rht** [comment]
Still needs to address https://github.com/projectmesa/mesa/pull/1898#issuecomment-1879649889.

**Reply by EwoutH** [reply]
Agreed, feel free to open PRs for them

**Comment by paulbanse** [comment]
I think it's overall a cool feature, it's a bit clunky to use and modify because it's model.grid.properties['nameofthelayer'].data[position] if you don't implement a shortcut, but it's very practical to have something dedicated for localized information.

**Reply by EwoutH** [reply]
Thanks for the feedback! Generally, I agree.

@quaquel did we already do this more elegant when coupled to a cell space?

**Reply by quaquel** [reply]
Yes, with cell spaces, you have attribute-like access. So `space.layer_name` (not 100% sure about this one) and `cell.layer_name` (100% sure this works - see the sugarscape example on read the docs).
